docs/octopus-rest-api/octopus-cli/run-runbook.md

```diff
@@ -0,0 +1,181 @@
+      --ignoreSslErrors      [Optional] Set this flag if your Octopus Server
+                             uses HTTPS but the certificate is not trusted on
+                             this machine. Any certificate errors will be
+                             ignored. WARNING: this option may create a
+                             security vulnerability.
+      --enableServiceMessages
+                             [Optional] Enable TeamCity or Team Foundation
+                             Build service messages when logging.
+      --timeout=VALUE        [Optional] Timeout in seconds for network
+                             operations. Default is 600.
+      --proxy=VALUE          [Optional] The URL of the proxy to use, e.g.,
+                             'https://proxy.example.com'.
+      --proxyUser=VALUE      [Optional] The username for the proxy.
+      --proxyPass=VALUE      [Optional] The password for the proxy. If both
+                             the username and password are omitted and
+                             proxyAddress is specified, the default
+                             credentials are used.
+      --space=VALUE          [Optional] The name or ID of a space within
+                             which this command will be executed. The default
+                             space will be used if it is omitted.
+      --keepalive=VALUE      [Optional] How frequently (in seconds) to send a
+                             TCP keepalive packet.
+      --logLevel=VALUE       [Optional] The log level. Valid options are
+                             verbose, debug, information, warning, error and
+                             fatal. Defaults to 'debug'.
+```
+
+## Basic Examples {#RunningRunbooks-Basicexamples}
+
+This will run a runbook that has a published snapshot (assuming an API key or username/password are set as environment variables):
+
+```bash
+octo run-runbook --runbook="Hello World"  \
+                 --project="Smurfs"       \
+                 --environment="Test"     \
+                 --server http://octopus/ \
+                 --apiKey API-ABCDEF123456
+```
+
+This will run a runbook using a specific snapshot (e.g. Snapshot KGHSL3L ):
+
+```bash
+octo run-runbook --runbook="Hello World" \
+                 --project="Smurfs"      \
+                 --environment="Test"    \
+                 --snapshot="Snapshot KGHSL3L"
+```
+
+To specify multiple environments, you can do:
+
+```bash
+octo run-runbook --runbook="Hello World" \
+                 --project="Smurfs"      \
+                 --environment="Test"    \
+                 --environment="Dev"
+```
+
+:::success
+You can run a runbook on ALL tenants in an environment by using the `--tenant=*` argument. This instructs Octopus.
+:::
+
+## Learn more
+
+- [Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md)
+- [Creating API keys](/docs/octopus-rest-api/how-to-create-an-api-key.md)
+- [Releases](/docs/managing-releases/index.md)
```

Comment:
Adding a link to the runbooks docs aswell:
```suggestion
- [Releases](/docs/managing-releases/index.md)
- [Runbooks](/docs/operations-runbooks/index.md)
```

Reply:
👍